test/Microsoft.AspNetCore.Razor.Language.Test/DefaultRazorIntermediateNodeLoweringPhaseIntegrationTest.cs

```diff
@@ -489,5 +491,29 @@ public void Lower_WithImports_IgnoresBlockDirective()
 
             return descriptor;
         }
+
+        private class DesignTimeOptionsFeature : IConfigureRazorParserOptionsFeature, IConfigureRazorCodeGenerationOptionsFeature
```

Comment:
sadface

Reply:
I had to do this because we don't really call `Process` or `ProcessDesignTime` here. We manually run each phase.

Reply:
I think you should keep going with this PR and we can discuss friday. I don't want to keep debating on this huge PR